app/views/administrate/application/_collection.html.erb

```diff
@@ -18,13 +18,13 @@ to display a collection of resources in an HTML table.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Table
 %>
 
-<table class="collection-data">
+<table class="collection-data" aria-labelledby="page-title">
```

Comment:
Do you think a `<caption>` might work better here? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption